src/effects/native/balanceeffect.cpp

```diff
@@ -0,0 +1,176 @@
+#include "balanceeffect.h"
+
+#include "util/defs.h"
+
+namespace {
+    double kMaxCorner = 22000; // Hz
```

Comment:
Oh, I understand now. IMO there is no use case for using the knob that way. If the user wants the effect to be bypassed they can turn the Balance parameter to center or turn the effect off.